### PR TITLE
Release 0.5.0

### PR DESCRIPTION
**Pull request by tjanc**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# 0.4.1 - 2017-08-08
+
+## Enhancements
+
+- Adds support for Fury 3.0.0-beta.4.
```
Comment (pksunkara) [inline CHANGELOG.md]: Please change this line to resemble something like https://github.com/apiaryio/fury-adapter-apib-parser/pull/17/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR3. Also, this is not an enhancement.

File: package.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
     "nunjucks": "^2.0.0"
   },
   "peerDependencies": {
-    "fury": "^3.0.0-beta.3"
+    "fury": "^3.0.0-beta.4"
```
Comment (pksunkara) [inline package.json:23]: There is a fury in devDependency too. Please update it.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "fury-adapter-apib-serializer",
-  "version": "0.4.1",
+  "version": "0.4.2",
```
Comment (kylef) [inline package.json]: https://github.com/apiaryio/fury-adapter-swagger/pull/118#discussion_r131921720

**Comment by pksunkara** [comment]
Closing because of #16

**Pull request closed without merging**